guide/english/working-in-tech/imposter-syndrome/index.md

```diff
@@ -37,6 +37,7 @@ More great resources to help you know more about Imposter Syndrome and some tips
 
 * [American Psychological Association - Feel Like a Fraud?](http://www.apa.org/gradpsych/2013/11/fraud.aspx)
 * [TED Talks - Fighting Imposter Syndrome](https://www.ted.com/playlists/503/fighting_impostor_syndrome)
+* [TED Talks - What is Imposter Syndrome and How Can You Combat It?] (https://www.ted.com/talks/elizabeth_cox_what_is_imposter_syndrome_and_how_can_you_combat_it?language=en)
```

Comment:
Needs formatting changes as mentioned in https://github.com/freeCodeCamp/freeCodeCamp/pull/23752#pullrequestreview-178584142